### ISSUE TITLE
Error on mouse hover over dead code

### ISSUE THREAD

**Issue by th3ragex** [microsoft/pyright#373]
**Describe the bug**
I get an error message in the VSCode Output window after trying to get an tooltip on 'inactive / dead' code. The error message is different in my big solution compared to my tiny sample provided below.

**To Reproduce**
![PyrightHoverError](https://user-images.githubusercontent.com/15040273/69792483-03e93280-11c7-11ea-990c-c264dd54fdae.gif)

![PyrightHoverError2](https://user-images.githubusercontent.com/15040273/69793079-131cb000-11c8-11ea-8b62-98af57b11305.gif)

**Expected behavior**
Everything seem to work, just the error message indicates something may be wrong.

**Screenshots or Code**
``` python
import numpy as np

def test():
    arr = np.array([1,2,3])
    return arr
    tmp = np.unique(arr)

```

> [Error - 10:03:21] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'id' of undefined
  Code: -32603 

> [Error - 10:13:23] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'lookUpSymbolRecursive' of undefined
  Code: -32603 

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool? 
-> Extension
Which version? 
-> 1.1.6

**Additional context**
VSCode About:
Version: 1.40.2 (system setup)
Commit: f359dd69833dd8800b54d458f6d37ab7c78df520
Date: 2019-11-25T14:54:45.096Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.17763

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of pyright.

**Comment by erictraut** [comment]
This is now fixed in 1.1.7, which I just published.
